biztalk/esb-toolkit/administration-using-the-esb-management-portal.md

```diff
@@ -55,6 +55,6 @@ The ESB Management Portal, included as part of the [!INCLUDE[esbToolkit](../incl
 
   -   Microsoft.Practices.Unity
 
-- [Microsoft Chart Controls](https://go.microsoft.com/fwlink/?LinkId=188293) (https://go.microsoft.com/fwlink/?LinkId=188293) for Microsoft .NET Framework 4
+- Microsoft Chart Controls for built in for .NET Framework 4
```

Comment:
"for built in for"?

Reply:
Ah typo, thank you. I've clarified and added a link for .NET 4.